app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -160,7 +161,7 @@ module.exports = React.createClass
           disabled = @getDisabledAttribute(i)
           selectedOption = if @state.selectedOptions[i]?.value then @state.selectedOptions[i] else null
           optionsKeys.push selectedOption.value if selectedOption?.value
-          <div key={select.id}>
+          <div id={select.id} key={select.id}>
```

Comment:
I think you can find a component by key. Something like `wrapper.find(div).find({key: select.id})`